static/js/util/profile_edit.js

```diff
@@ -35,7 +35,14 @@ export function boundRadioGroupField(keySet: string[], label: string, options: O
 
   let onChange = e => {
     let clone = _.cloneDeep(profile);
-    _.set(clone, keySet, e.target.value);
+    let value = e.target.value;
+    // string to boolean conversion.
```

Comment:
this comment isn't necessary, this code is very straightforward and understandable on it's own